synapseclient/table.py

```diff
@@ -654,9 +698,12 @@ class ViewBase(SchemaBase):
         "ignoredAnnotationColumnNames",
     ]
 
-    def add_scope(self, entities):
+    def add_scope(self, entities: Union[Entity, list, str]):
         """
-        :param entities: a Project, Folder, Evaluation object or its ID, can also be a list of them
+        Add scope
+
+        Arguments:
+            entities: a Project, Folder, Evaluation object or its ID, can also be a list of them
```

Comment:
If we are accepting these types (`Project`, `Folder`, `Evaluation`) could you update the `entities` typing to reflect that.

Also, it might be worthwhile to link to those classes here. ie:

`a [Project][synapseclient.Project]...`

Reply:
Done